src/components/TubeMapContainer.js

```diff
@@ -28,7 +28,7 @@ class TubeMapContainer extends Component {
         ? this.getRemoteTubeMapData()
         : this.getExampleData();
     } else {
-      if (JSON.stringify(this.props.fetchParams) !== JSON.stringify(prevProps.fetchParams)) {
+      if (JSON.stringify(this.props.viewTarget) !== JSON.stringify(prevProps.viewTarget)) {
```

Comment:
Since we now have `react-fast-compare` in the project, we should use that here instead of JSON stringification.